views/eApostilles/additionalReference.ejs

```diff
@@ -38,12 +40,12 @@
                     You can also add your own reference here if you want to.
```

Comment:
Would it be worth putting some copy here to explain the characters that are  valid and accepted?

Reply:
That's a good point I will speak to Liz about this.